hopsworks-common/src/main/java/io/hops/hopsworks/common/util/ProjectUtils.java

```diff
@@ -60,20 +60,41 @@ public boolean isReservedProjectName(String projectName) {
     }
     return false;
   }
+    
+  public String getFullDockerImageName(Project project){
+    return getFullDockerImageName(project, settings);
+  }
   
-  public String getCurrentCondaEnvironment(Project project) {
-    String condaEnv = project.getName();
+  public static String getFullDockerImageName(Project project, Settings settings){
```

Comment:
We don't need to pass the Settings bean here, it's better to declare as a class variable like the other beans